internals/cli/cmd_check.go

```diff
@@ -62,6 +70,41 @@ func (cmd *cmdCheck) Execute(args []string) error {
 		return ErrExtraArgs
 	}
 
+	if cmd.Refresh {
+		opts := client.RefreshCheckOptions{
+			Name: cmd.Positional.Check,
+		}
+		res, err := cmd.client.RefreshCheck(&opts)
+		if err != nil {
+			return err
+		}
+
+		checkInfo := checkInfo{
+			Name:      res.Info.Name,
+			Level:     string(res.Info.Level),
+			Startup:   string(res.Info.Startup),
+			Status:    string(res.Info.Status),
+			Failures:  res.Info.Failures,
+			Threshold: res.Info.Threshold,
+			ChangeID:  res.Info.ChangeID,
+		}
+		if res.Error != "" {
+			checkInfo.Error = res.Error
+			logs, err := cmd.taskLogs(checkInfo.ChangeID)
```

Comment:
`ChangeID` can be empty in some edge cases, so we should only call this if `ChangeID != ""`. See similar code in `cmdChecks.changeInfo`.